docs/docs/how-to-deploy.mdx

```diff
@@ -53,6 +53,275 @@ However, you can also customize the Helm charts if you have specific requirement
+[Microsoft Azure](https://azure.microsoft.com/en-us/services/kubernetes-service/), or
+[Amazon EKS](https://aws.amazon.com/eks/).
+
+:::note
+The Rasa Helm chart is open-source and available in the
+[helm-charts repository](https://github.com/rasahq/helm-charts).
+Please
+[create an issue](https://github.com/RasaHQ/helm-charts/issues/new) in this
+repository if you discover bugs or have suggestions for improvements.
+
+:::
+
+
+### Installation Requirements
+
+1. Please check that you installed the Kubernetes or OpenShift command line
+   interface (CLI). You can check this using the following command:
+
+   <Tabs values={[{"label": "Kubernetes", "value": "kubernetes"}, {"label": "OpenShift", "value": "openshift"}]} defaultValue="kubernetes">
+     <TabItem value="kubernetes">
+
+     ```bash
+     kubectl version --short --client
+
+     # The output should be similar to this
+     # Client Version: v1.19.11
+     ```
+
+     </TabItem>
+     <TabItem value="openshift">
+
+     ```bash
+     oc version --client
+
+     # The output should be similar to this
+     # Client Version: 4.7.13
+     ```
+
+     </TabItem>
+   </Tabs>
+
+   If this command resulted in an error, please install the
+   [Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/) or the
+   [OpenShift CLI](https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli)
+   depending on the cluster you’re using.
+
+2. Make sure that the Kubernetes / OpenShift CLI is correctly connected to
+   your cluster. You can do so by using the following commands:
+
+   <Tabs values={[{"label": "Kubernetes", "value": "kubernetes"}, {"label": "OpenShift", "value": "openshift"}]} defaultValue="kubernetes">
+     <TabItem value="kubernetes">
+
+     ```bash
+     kubectl version --short
+
+     # The output should be similar to this
+     # Client Version: v1.19.11
+     # Server Version: v1.19.10
+     ```
+
+     </TabItem>
+     <TabItem value="openshift">
+
+     ```bash
+     oc version
+
+     # The output should be similar to this
+     # Client Version: 4.7.13
+     # Kubernetes Version: v1.20.0+df9c838
+     ```
+
+     </TabItem>
+   </Tabs>
+
+   If you get an error when executing the command, you are not connected to your
+   cluster. To get the command to connect to the cluster please consult your cluster’s
+   admin or the documentation of your cloud provider.
+
+3. Please make sure you have the [Helm CLI](https://helm.sh/docs/intro/install/)
+   installed. To check this, run:
+
+   ```bash
+   helm version --short
+
+   # The output should be similar to this
+   # v3.6.0+g7f2df64
+   ```
+
+   If this command leads to an error, please install the
+   [Helm CLI](https://helm.sh/docs/intro/install/).
+
+   In case you are using a version `<3.5` of Helm, please update to Helm version
+   `>=3.5`.
+
+### Installation
+
+#### 1. Create Namespace
+
+We recommend installing Rasa Assistant in a separate
```

Comment:
```suggestion
We recommend installing the Rasa Open Source server in a separate
```

Reply:
Or maybe just "Rasa Open Source" (without server?)

Reply:
"Rasa Open Source" sounds good to me 😄